File: pgm/ABC_Java/ABC_419_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S = reader.readLine();
        reader.close();
        final String red = "red";
        final String blue = "blue";
        final String green = "green";
        final String SSS = "SSS";
        final String FFF = "FFF";
        final String MMM = "MMM";
        final String Unknown = "Unknown";
        final Map<String, String> language =
            Map.of (red, SSS, blue, FFF, green, MMM);
        System.out.println (language.getOrDefault (S, Unknown));
        System.exit (0);
    }
}

File: pgm/ABC_Kotlin/ABC_419_A.kt
// import java.time.*

// import kotlin.math.*

const val red: String = "red"
const val blue: String = "blue"
const val green: String = "green"
const val SSS: String = "SSS"
const val FFF: String = "FFF"
const val MMM: String = "MMM"
const val Unknown: String = "Unknown"

fun main() {
    val S: String = readln()
    val Language: Map<String, String> = mapOf(red to SSS, blue to FFF, green to MMM)
    if (S in Language) {
        println(Language[S])
    } else {
        println(Unknown)
    }
    kotlin.system.exitProcess(0)
}
